packages/react-component-library/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Modal } from '../Modal'
+import { ButtonProps } from '../Button'
+
+export interface DialogProps {
+  title?: string
+  description?: string
+  danger?: boolean
+  onConfirm?: (event: React.SyntheticEvent) => void
+  onCancel?: (event: React.SyntheticEvent) => void
+  isOpen?: boolean
+}
+
+export const Dialog: React.FC<DialogProps> = ({
+  title,
+  description,
+  danger = false,
+  onConfirm,
+  onCancel,
+  ...rest
+}) => {
+  const confirmButton: ButtonProps = {
+    onClick: onConfirm,
+    children: 'Confirm',
+    variant: 'primary',
+    color: danger ? 'danger' : undefined,
+  }
+
+  const cancelButton: ButtonProps = {
+    onClick: onCancel,
+    children: 'Cancel',
+    variant: 'tertiary',
+  }
+
+  return (
+    <Modal
+      className={`rn-dialog ${danger ? 'rn-dialog--danger' : ''}`}
```

Comment:
Take a peek in the repo for uses of 'classNames'. It's an alternative way to specify conditional classes without all the extra spaces that can creep in.

Reply:
Yeah ok I used this in the button fixes.